src/Apps/Artwork/Components/ArtworkBanner/index.tsx

```diff
@@ -0,0 +1,155 @@
+          />
+        )
+      case "ArtworkContextFair":
+        const fairImage = get(context, c => c.profile.icon.img.url)
+        const initials = get(context, c => c.profile.initials)
+        return (
+          <Banner
+            imageUrl={fairImage}
+            initials={initials}
+            badge="At fair"
+            // headline={context.name}
+            headline="Fair name goes here"
+            subHeadline={partner.name}
+          />
+        )
+      case "ArtworkContextPartnerShow":
+        const showImage = get(context, c => c.thumbnail.img.url)
+        let showLine = "In current show"
+        if (context.status === "upcoming") {
+          showLine = "In upcoming show"
+        } else if (context.status === "closed") {
+          showLine = "In past show"
+        }
+        return (
+          <Banner
+            imageUrl={showImage}
+            initials={partner.initials}
+            badge={showLine} // headline={context.name}
+            headline="Show name goes here"
+            subHeadline={partner.name}
+          />
+        )
+      default:
+        return null
+    }
+  }
+}
+
+export const ArtworkBannerFragmentContainer = createFragmentContainer(
+  ArtworkBanner,
+  graphql`
+    fragment ArtworkBanner_artwork on Artwork {
+      context {
+        __typename
+        ... on ArtworkContextAuction {
+          name
+          href
+          is_auction
+          is_closed
+          is_open
+          live_start_at
+          live_url_if_open
+        }
+        ... on ArtworkContextFair {
+          name
+          href
+          is_active
+          start_at
+          end_at
+          profile {
+            initials
+            icon {
+              img: resized(width: 70, height: 70, version: "square") {
+                url
+              }
+            }
+          }
+        }
+        ... on ArtworkContextPartnerShow {
+          name
+          href
+          type
+          status
+          thumbnail: cover_image {
+            img: resized(width: 70, height: 70, version: "square") {
+              url
+            }
+          }
+        }
+      }
+      partner {
+        type
+        name
+        initials
+        profile {
+          icon {
+            url(version: "square140")
+          }
+        }
+      }
+    }
+  `
+)
+
+export const ArtworkBannerQueryRenderer = ({
+  artworkID,
+}: {
+  artworkID: string
+}) => {
+  return (
+    <ContextConsumer>
+      {({ user, mediator, relayEnvironment }) => {
+        return (
+          <QueryRenderer<ArtworkBannerQuery>
+            environment={relayEnvironment}
+            variables={{ artworkID }}
+            query={graphql`
+              query ArtworkBannerQuery($artworkID: String!) {
+                artwork(id: $artworkID) {
+                  ...ArtworkBanner_artwork
+                }
+              }
+            `}
+            render={renderWithLoadProgress(
+              ArtworkBannerFragmentContainer as any
```

Comment:
Also again will not render for me locally without casting to `any` here. Might be able to remove that once I figure out this name thing. I home:)

Reply:
Yeah this is a thing on your env, because it works fine again for me when I remove it and also last time when I PR’ed removing them CI was fine too. Let’s do a quick pair on this.